tasks/create-advisory/create-advisory.yaml

```diff
@@ -65,6 +65,15 @@ spec:
         advisoryData=$(jq -c "$(params.jsonKey)" $(workspaces.data.path)/$(params.dataPath))
         configMapName=$(jq -er '.sign.configMapName' $(workspaces.data.path)/$(params.dataPath))
 
+        # Validate type
+        if jq -e "$(params.jsonKey).type" $(workspaces.data.path)/$(params.dataPath) > /dev/null 2>&1; then
+          advisoryType=$(jq -c "$(params.jsonKey).type" $(workspaces.data.path)/$(params.dataPath))
+          if ! [[ "advisoryType" =~ ^(RHSA|RHBA|RHEA)$ ]]; then
```

Comment:
Missing `$` in `"$advisoryType"`. How is your test working without it?

Reply:
Because it is allowed to fail, which it does. Anyway, fixed.